compiler/quilt/tools/build.py

```diff
@@ -527,6 +527,8 @@ def load_yaml(filename, optional=False):
         data = fd.read()
     try:
         res = yaml.load(data)
+        if not any(k in res.keys() for k in ['contents', 'packages']):
+            raise BuildException('"contents" or "packages" node must be specified')
```

Comment:
Let's break this out into two cases:
* `Error in FILENAME: missing 'contents' node`
* `Error in FILENAME: missing 'pacakges' node`

Reply:
Also note, 'packages' is an entirely different use case. That's only for `quilt.yml`.